### PR TITLE
Disable domain block

### PR DESCRIPTION
**Pull request by iefremov**

### CODE REVIEW

**Comment by iefremov** [comment]
@kjozwiak @LaurenWags @bbondy FYI

File: seed/seed.json [hunk 1]
```diff
@@ -1,6 +1,34 @@
...
+                        "disable_feature": ["BraveDomainBlock"]
+                    }
+                },
+                {
+                    "name": "Default",
+                    "probability_weight": 0
+                }
+            ],
+            "filter": {
+                "max_version": "89.1.23",
```
Comment (iefremov) [inline seed/seed.json:27]: this is inclusive

File: seed/seed.json [hunk 2]
```diff
@@ -1,6 +1,34 @@
...
+                "channel": ["RELEASE"],
```
Comment (iefremov) [inline seed/seed.json:28]: only stable channel is affected

**Review by moritzhaller — APPROVED** [review]
++

**Review by kjozwiak — APPROVED** [review]
LGTM 👍 QA went through the following on staging as per https://github.com/brave/brave-browser/issues/15149#issuecomment-816936276. BTW, can't merge as I don't have `write` access to https://github.com/brave/brave-variations

**Comment by kjozwiak** [comment]
@iefremov we'll want to revert this as we're going via the PR route via https://github.com/brave/brave-core/pull/8507.

**Pull request merged**